packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -75,6 +75,14 @@ export const messages = {
   parameter_gatherer_invalid_forceide_url:
     "The forceide:// URL is invalid. From your subscriber's org, copy and paste the forceide:// URL shown on the Apex Debugger page in Setup.",
   parameter_gatherer_enter_function: 'Enter function details',
+  parameter_gatherer_prompt_confirm_option: 'Continue',
```

Comment:
@ekapner these labels will be used to confirm/cancel the execution of the new commands

![Screen Shot 2020-09-14 at 3 00 58 PM](https://user-images.githubusercontent.com/1041842/93143229-5cc58100-f69c-11ea-962f-c7950547181c.png)

![Screen Shot 2020-09-14 at 3 00 38 PM](https://user-images.githubusercontent.com/1041842/93143243-6222cb80-f69c-11ea-9923-8e9b0afac095.png)

![Screen Shot 2020-09-14 at 3 00 23 PM](https://user-images.githubusercontent.com/1041842/93143251-66e77f80-f69c-11ea-8dd0-f580feddd3ef.png)

Reply:
For second/middle - should we include "deleted"?
Confirm to continue removing deleting and expired scratch orgs (?)

Reply:
Will do 👍